### PR TITLE
[1-24] Header component creation

### PR DESCRIPTION
**Pull request by mityajs**
Header component creation

### CODE REVIEW

File: exalearn-ui/src/index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
 		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
 	</head>
 	<body class="mat-typography">
+    <header></header>
 		<app-root></app-root>
 	</body>
```
Comment (KuznetsovR) [inline exalearn-ui/src/index.html:16]: All components and everything else should be in app-root, also usually components have the 'app-' prefix in selector.

File: exalearn-ui/src/app/header/header.component.less [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.header {
+  position: fixed;
+  width: 100vw;
+  background-color: #e9e9e9;
```
Comment (YanaBr) [inline exalearn-ui/src/app/header/header.component.less]: Please, change .less to .scss and refactor styles in according to this format.
Note: After @GeranSkot will merge his PR, all colors we have to replace by constants from styles/colors.scss

File: exalearn-ui/src/app/header/header.component.less [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+  .logo {
+    position: absolute;
+    left: 10px;
+    top: 10px;
+    width: 48px;
+    height: 40px;
+  }
+
+  div {
```
Comment (YanaBr) [inline exalearn-ui/src/app/header/header.component.less]: Really bad practice to write tag-name selector. Please, try to replace it with .class-name selector (or maybe #id-selector)

File: exalearn-ui/src/app/header/header.component.less [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    display: inline-flex;
+
+    div {
```
Comment (YanaBr) [inline exalearn-ui/src/app/header/header.component.less]: Same here

File: exalearn-ui/src/app/header/header.component.less [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+      margin-right: 10px;
+      height: 40px;
+      line-height: 40px;
+      border-radius: 20px;
+      padding-left: 10px;
+      padding-right: 10px;
+    }
+
+    div:hover {
```
Comment (YanaBr) [inline exalearn-ui/src/app/header/header.component.less]: Same here

File: exalearn-ui/src/index.html (hunk 1 above)
Comment (KuznetsovR) [inline exalearn-ui/src/index.html]: All components and everything else should be in app-root, also usually components have the 'app-' prefix in selector.

File: exalearn-ui/src/app/header/header.component.less [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+      cursor: pointer;
+    }
+
+    .profile {
+      background-color: white;
```
Comment (YanaBr) [inline exalearn-ui/src/app/header/header.component.less]: Same about color, please, wait a bit PR with colors

File: exalearn-ui/src/app/header/header.component.ts [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  selector: 'header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.less']
+})
+export class HeaderComponent {
+  isAppInitialized = true;
+  isAuthorized = true;
+
+  public profile() {
+    console.log('profile_click');
```
Comment (YanaBr) [inline exalearn-ui/src/app/header/header.component.ts]: Please, remove console.logs from PR

**Review by KotdenGit — APPROVED** [review]

**Review by KotdenGit — APPROVED** [review]

File: exalearn-ui/src/index.html (hunk 1 above)
Comment (YanaBr) [reply exalearn-ui/src/index.html]: For now nothing should in AppComponent, header/footer/etc. we place on the MainPageComponent

**Review by YanaBr — CHANGES_REQUESTED** [review]
Please, pull changes from dev and use common constants

**Review by YanaBr — APPROVED** [review]

**Pull request merged**